WebUI/build/README.md

```diff
@@ -1,26 +1,18 @@
 ## Prepare a base python 3.11 environment
 
-1. Download embedded python for windows from https://github.com/adang1345/PythonWindows
-2. Download get-pip.py from https://bootstrap.pypa.io/get-pip.py
-3. Install miniforge: https://github.com/conda-forge/miniforge
-4. Create a reference conda environment with libuv installed
-5. Download 7zr executable from https://www.7-zip.org/a/7zr.exe, put it under `WebUI\package_res` folder.
-
-```
-conda create -n cp311_libuv python=3.11 libuv -y
-
-# copy the path to this conda env
-conda env list | findstr cp311_libuv
-```
-
-5. Run pack-python script with 3 additional arguments, this will generate `env.7z` under `WebUI\package_res` folder.
-
-```
-cd WebUI
-npm run pack-python <python_embed_zip> <get_pip_py> <ref_conda_env>
-```
-
-`package_res/env.7z` could be reused for all platforms.
+1. fetch resources via web by calling ```npm run fetch-build-resources```
+2. provide windows libuv dlls:
+   - Install miniforge: https://github.com/conda-forge/miniforge
+   - Create a reference conda environment with libuv installed
+    ```
+    conda create -n cp311_libuv python=3.11 libuv -y
+    # copy the path to this conda env
+    conda env list | findstr cp311_libuv
+    ```
+    - symlink or copy the conda env path into /python_package_resources
+3. run ```npm run pack-python```
+
+The resulting `WebUI/npm_package_res/env.7z` could be reused for all platforms.
 
```

Comment:
In what circumstance does this need to be executed and reference to GitHub action

Reply:
I am not sure, if I understand the question. Do you mean, what we do with the env.7z?

This packaged empty is "as is" newly setup python env. It awaits the correct pip install commands to run to enrich for the platform specific) dependencies so the app can bootup the backend. As there are no dependencies installed, this env.7z can be used for any of the builds : arc/ultra/ultra2 and does not need to be rebuilt in general.

The npm build scripts expect this to be present, and so does the github action (at the correct point in time). The build scripts will fail, if it is not present